eventProcessor/app.js

```diff
@@ -24,13 +27,45 @@ const lambdaInvoke = (issuesChunks) => {
   return lambda.send(command);
 };
 
+const axiosClient = axios.create({
+  baseURL: 'https://heapanalytics.com/api',
+  headers: {
+    'Content-Type': 'application/json',
+    accept: 'application/json',
+  }
+});
+
+const sendCallHomeData = async (callHomePayload) => {
```

Comment:
This is ok for now. We should extract this into its own file/module similar to logger.

Reply:
will do in the next PR